book-3-workforce/chapters/VIEWMODELS.md

```diff
@@ -170,25 +226,43 @@ namespace StudentExercises.Models.ViewModels
 
         public StudentCreateViewModel(IConfiguration config)
```

Comment:
Remove the `config` param and add a connection string param.

Reply:
Removed